src/parser/cvc/Cvc.g

```diff
@@ -1683,12 +1683,12 @@ uminusTerm[CVC4::Expr& f]
   unsigned minusCount = 0;
 }
     /* Unary minus */
-  : (MINUS_TOK { ++minusCount; })* bvBinaryOpTerm[f]
+  : (MINUS_TOK { ++minusCount; })* bvBinaryOp[f]
```

Comment:
Can you revert the changes in this file? They don't have anything to do with your changes.

Reply:
Oops, sure! Done with a0f690b64f5f9fd921cf3f8313e980134accda8e. I assumed it was talking about the same "OpTerm" -- my bad.

Reply:
I guess this was coincidence :-)

It should in the future, but we shouldn't include it in this PR (but rather do this explicitly when migrating the parser).